js/dist/wp-seo-admin-global-320.js

```diff
@@ -136,7 +136,9 @@
 	function hookDismissRestoreButtons() {
 		var $dismissible = $( '.yoast-alert-holder' );
 
-		$dismissible.on( 'click', '.dismiss', function() {
+		$dismissible.on( 'click', '.dismiss', function( evt ) {
+			evt.preventDefault();
```

Comment:
If it is a button, why do we need an `evt.preventDefault`? If the problem was that the form would be submitted that can be fixed using `<button type="button">`.